src/components/SchematixFloorplan.vue

```diff
@@ -41,7 +41,7 @@ const details = [
   { label: 'Alfresco', value: props.item.alfrescoArea, suffix: 'm<sup>2</sup>' }
 ].filter(({ value }) => value != 0)
 
-const remainingDetails = details.length % 3
+const remainingDetails = details.length - (details.length % 3) - 1
```

Comment:
Crappy variable name fyi. In future I suggest you try to think of a name that actually describes what this is for.